tests/integration/bond_test.py

```diff
@@ -353,6 +355,73 @@ def test_changing_slave_order_keeps_mac_of_existing_bond(bond99_with_2_slaves):
     )
 
 
+@pytest.mark.tier1
+def test_adding_a_slave_keeps_mac_of_existing_bond(bond99_with_eth2, eth1_up):
+    bond_state = bond99_with_eth2[Interface.KEY][0]
+    bond_slaves = bond_state[Bond.CONFIG_SUBTREE][Bond.SLAVES]
+    bond_slaves.insert(0, ETH1)
+
+    current_state = statelib.show_only((BOND99,))
+
+    libnmstate.apply(bond99_with_eth2)
```

Comment:
This is very confusing when reading. It is no longer a bond with one slave. Maybe do an assignment at the top: `desired_state = bond99_with_eth2`.

Reply:
Done!